cake_thief/robertwpaul/cake_thief.rb

```diff
@@ -0,0 +1,13 @@
+Cake = Struct.new(:weight, :value)
+
+def sorted_cakes(cakes)
+  cakes.sort { |a, b| b.value / b.weight <=> a.value / a.weight }
```

Comment:
First time I've seen this operator, nice. Today I learned about the _spaceship_ ruby operator 👍